### ISSUE TITLE
The UrlInput component is not accessible

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#1855]
The UrlInput component added in https://github.com/WordPress/gutenberg/commit/4dc462c9a8c73a4c7a2e4fe4a6b6fe1cb73e3b52 to support optional links in image blocks, is made of two buttons and one input field:

<img width="398" alt="screen shot 2017-07-11 at 20 22 49" src="https://user-images.githubusercontent.com/1682452/28084036-b265ef68-6677-11e7-9fce-acd91a630fce.png">

The two buttons are "empty": there's no text or aria-label that assistive technologies can announce. Screen reader will read out just "button".

A very basic requirement for web standards and accessibility is that **all user interface controls need to be labeled.**

In this case, an `aria-label` with a meaningful text would be ok.

The input field misses a properly associated label. All input fields need to be associated with a label, that can be visually hidden. Alternatively, they can use an aria-label attribute. [Placeholders are not a replacement for labels](https://core.trac.wordpress.org/ticket/40331).

**Comment by afercia** [comment]
Fixed in #1856
